profiles/common/modules/custom/multisite_create_button/multisite_create_button.install

```diff
@@ -9,9 +9,7 @@
  * Implements hook_enable().
  */
 function multisite_create_button_enable() {
```

Comment:
Ok, but why the multisite_create_button_disable is still in charge of remove it then?